vars/fhBuildNode.groovy

```diff
@@ -2,9 +2,10 @@
 
 def call(Map parameters = [:], body) {
 
-    def label = parameters.get('label', 'nodejs-ubuntu')
+    def labels = [params0.get('label')].findAll {it} parameters.get('labels',[]) + ['nodejs'] + ['ubuntu']
+    labels += [params.get('label')].findAll {it}  
```

Comment:
Ok, I have decided, that the de-duplicating and null removing will be done after you call the withLabels function